modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * installing/installing_azure/installing-azure-private.adoc
+// * installing/installing_azure/installing-azure-vnet.adoc
+
+:_content-type: PROCEDURE
+[id="finalizing-encryption_{context}"]
+= Finalizing user-managed encryption after installation
+If you installed {product-title} using a user-managed encryption key, you can complete the installation by updating the default storage class and granting `Owner` permissions to the Azure cluster resource group.
```

Comment:
"Owner" isn't a requirement. A role that has write permissions to the disk encryption set will be necessary however.

Reply:
I have updated the "finalizing" module to take this feedback into account. PTAL

Reply:
I updated the privileged role annotation to state "Specifies an Azure role that has read/write permissions to the disk encryption set. You can use the `Owner` role or a custom role with the necessary permissions." Does this sound better to you?

Reply:
Sounds good to me